### PR TITLE
Apply patch to work with NGINX v1.23

### PR DESCRIPTION
**Pull request by volks73**
Applied [this patch](https://github.com/masterzen/nginx-upload-progress-module/files/8980323/nginx_1.23.0.patch.txt) to add compatibility with NGINX v1.23.

### CODE REVIEW

**Comment by volks73** [comment]
@Theia-Scientific/developers Ready to review.

**Comment by mikhail-vl** [comment]
@volks73 I would suggest keeping the original formatting. It will be hard to follow mainstream changes with different formatting.

**Comment by mikhail-vl** [comment]
@volks73 In the issue https://github.com/masterzen/nginx-upload-progress-module/issues/56 there is a patch to 1.24 if we want to use it instead.

**Comment by volks73** [comment]
It was my autoformatter.

**Comment by volks73** [comment]
This was resolved by applying the patch in our internal repository.
